### ISSUE TITLE
Maximize screen space usage

### ISSUE THREAD

**Issue by Skasi** [jaredforrest/nonocross#5]
Sometimes tiles can be very small and thus difficult to tap with a finger. To fix this, puzzles should be rotated by 90° to maximize tilesize, if viable.

At the moment we can increase the width of puzzles as much as we like. But the height is limited by the width. However, in portrait mode the screen is higher than it is wide, so it should be the other way around.

This ticket is not asking for widescreen mode, but since puzzles can be rotated by 90° without changing anything about the puzzle, widescreen mode would only need to rotate the numbers by 90° and not the tiles themself.

TL;DR: The game should look like on the left side instead of the right side:
![grafik](https://user-images.githubusercontent.com/132906/106182546-475e7880-619f-11eb-80fd-ce67563af878.png)

I'm not sure, but this could be as simple as swapping the `setOnValueChangedListener`s like this:
https://github.com/jaredforrest/nonocross/commit/5b57393b0fb12d570ce7c05f9f86afd29e5cae09 (untested, hence no PR)

**Comment by jaredforrest** [comment]
Sorry I'm quite busy at the moment so I can't really take a good look at this issue, but I think your change may cause the puzzle to extend beyond the screen height. Even if that change doesn't work this is something that should be fixed so when I get some free time I'll take a look (feel free to submit a PR if you've tested it)

**Comment by hcgrove** [comment]
I basically never choose a puzzle greater than 6x6 as the tiles become too small to hit comfortably on my phone, and I skipped the last practice board, as I on the second-to-last felt that I spend more time on correcting wrong hits, than I spend on actually solving the puzzle. None of that can be intended.

**Comment by jaredforrest** [comment]
I've fixed this but I'm not sure how the f-droid auto updater works so it may take a few days to update in f-droid

**Comment by jaredforrest** [comment]
The update has been pushed through on fdroid, so I'm closing the issue
